backend/package.json

```diff
@@ -6,7 +6,8 @@
   "type": "module",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "start": "nodemon server.js",
```

Comment:
This affects the deployment, 
Revert starts as it was i.e node server.js
`dev` can be used to run with nodemon but revert the `start` command